cmrtlib/linux/CMakeLists.txt

```diff
@@ -90,8 +91,26 @@ endif()
 set_target_properties( igfxcmrt PROPERTIES 
   FOLDER CM_RT 
   COMPILE_DEFINITIONS "${CMRT_DEFINES}" 
-  OUTPUT_NAME "igfxcmrt${TARGET_MODIFIER}"
+  OUTPUT_NAME "libigfxcmrt${TARGET_MODIFIER}"
```

Comment:
"lib" looks strange. I think you need to remove it from here + remove set(PREFIX "") below. Linux will do the trick for you.

Reply:
I did fix it in my last update. Can you review it in the latest file changes?